src/api/teams/create.rs

```diff
@@ -0,0 +1,66 @@
+use super::*;
+use crate::params;
+
+#[derive(serde::Serialize)]
+pub struct CreateTeamBuilder<'octo, 'a> {
+    #[serde(skip)]
+    handler: &'a TeamHandler<'octo>,
+    name: String,
+    description: Option<String>,
+    maintainers: Option<Vec<String>>,
+    repo_names: Option<Vec<String>>,
+    privacy: Option<params::teams::Privacy>,
+    permission: Option<params::teams::Permission>,
+    parent_team_id: Option<i64>,
+}
+
+impl<'octo, 'a> CreateTeamBuilder<'octo, 'a> {
+    pub(crate) fn new(handler: &'a TeamHandler<'octo>, name: String) -> Self {
+        Self {
+            handler,
+            name,
+            description: None,
+            maintainers: None,
+            repo_names: None,
+            privacy: None,
+            permission: None,
+            parent_team_id: None,
+        }
+    }
+
+    pub fn description(mut self, description: impl Into<String>) -> Self {
+        self.description = Some(description.into());
+        self
+    }
+
+    pub fn maintainers(mut self, maintainers: impl Into<Vec<String>>) -> Self {
+        self.maintainers = Some(maintainers.into());
+        self
+    }
+
+    pub fn repo_names(mut self, repo_names: impl Into<Vec<String>>) -> Self {
+        self.repo_names = Some(repo_names.into());
+        self
+    }
+
+    pub fn privacy(mut self, privacy: impl Into<params::teams::Privacy>) -> Self {
+        self.privacy = Some(privacy.into());
+        self
+    }
+
+    #[deprecated = "This has been deprecated by GitHub."]
+    pub fn permission(mut self, permission: impl Into<params::teams::Permission>) -> Self {
+        self.permission = Some(permission.into());
+        self
+    }
+
```

Comment:
I would prefer not to include anything that has already been deprecated by GitHub.
```suggestion
```